src/views/UpdatesFeed/UpdatesFeed.js

```diff
@@ -90,7 +90,10 @@ export default class UpdatesFeed extends React.Component {
             cells={
               updates
                 .slice(1, CONSTANTS.NUM_UPDATES_SHOWN).map((update, i) => (
-                  <Update update={update} key={i} />
+                  <FeaturedUpdate
```

Comment:
Using FeaturedUpdate for all updates for now until we have more udpates to display here.